packages/ipfs/README.md

```diff
@@ -942,6 +942,29 @@ const node = await IPFS.create({
 
 The code above assumes you are running a local `signaling server` on port `9090`. Provide the correct values accordingly.
 
+**Note:** We host a signalling server at star-signal.cloud.ipfs.team that can be used for practical demos and experimentation, it should not be used for apps in production. A libp2p-webrtc-star address, using the signalling server we provide, looks like:
+
+/dns4/star-signal.cloud.ipfs.team/wss/p2p-webrtc-star/p2p/<your-peer-id>
+
+Note: The address above indicates WebSockets Secure, which can be accessed from both http and https.
+
+#### Is there a more stable alternative to webrtc-star that offers a similar functionality?
```

Comment:
I don't think this is an accurate statement to make. It is an alternative but unless I am mistaken we have no performance measurements between the two to say stardust is more/less stable. Ideally, we should do some load testing on the rendezvous servers to make these comparisons.